internal/cmd/command.go

```diff
@@ -145,7 +146,20 @@ func NewConfluentCommand(cfg *v1.Config) *cobra.Command {
 	return cmd
 }
 
-func Execute(cmd *cobra.Command, args []string, cfg *v1.Config) error {
+func Execute(cmd *cobra.Command, args []string, cfg *v1.Config) (err error) {
+	defer func() {
+		if r := recover(); r != nil {
+			//TODO: need to uncomment before releasing
+			// if !cfg.Version.IsReleased() {
+			//	panic(r)
+			// }
+			err = &ppanic.Panic{ErrorMsg: ppanic.FormatPanicMsg(r)}
```

Comment:
This might be the more idiomatic way to do this.
```suggestion
			err = ppanic.NewError(r)
```

Reply:
https://pkg.go.dev/github.com/spf13/cobra#CheckErr makes it so we don't have to manually add "Error:" and it also automatically exits with code 1.